changelog

```diff
@@ -1,3 +1,142 @@
+Version 0.10.9
+  * Big changes:
+    * Added an alien faction dwelling in a gas giant in the galactic Northeast. (@RisingLeaf, @GefullteTaubenbrust2)
+    * Added a faction of amphibious alien to the galactic south. (@Daeridanii1)
+    * Added Corroboration I, a new mission chain for the Gegno. (@Saugia)
+    * When attempting to capture a ship, an attacker's "attack" rating will always be pitted against the opposition "defense" rating, instead of only when the opposition chooses to defend. (@petervdmeer, @Hurleveur)
```

Comment:
```suggestion
    * During hand to hand combat, if both sides choose to attack, two rounds of combat are run, one with the player's attack against the enemy's defense and one with the enemy's attack against the player's defense, instead of using the player's attack against the enemy's attack. (@petervdmeer, @Hurleveur)
```

Reply:
Might this add too many instances of "attack"?

Reply:
This is the changelog rather than a piece of story writing, so I think it's better that it provides an accurate description of what was changed even if it's not a particularly elegant piece of prose.